server/src/controllers/campaign.js

```diff
@@ -0,0 +1,97 @@
+const db = require("../../database/models/index");
+const BadRequestError = require("../errors/bad-request-error");
+const Op = db.Sequelize.Op;
+
+class CampaignController {
+  static getCampaigns = async (req, res) => {
+    const user = await db.User.findOne({
+      where: {
+        id: req.params.userId
+      }
+    });
+
+    if (!user) {
+      throw new BadRequestError("User does not exist");
+    }
+
+    const campaigns = await user.getCampaigns();
+
+    res.send(campaigns);
+  };
+
+  static getCampaign = async (req, res) => {
```

Comment:
We should only allowed the logged in user to get their campaign

Reply:
Updated to get only the logged in user's campaign data